### DISCUSSION TITLE
index.html in locale folder (noExtraDir: true)

### DISCUSSION THREAD

**Discussion by shishkin17** [vikejs/vike#1760] [Discussion] [answered]
When enabling `noExtraDir: true`, it generates **fr.html** (for French locale) instead of **fr/index.html**.

It would be better to generate the second variant as for me. What do you think?

**Comment by shishkin17** [comment, marked as answer]
I added `export default '/index'` to +route.ts for index page and it worked)

**Reply by brillout** [reply]
I'm not sure I understand what the problem was and this solves it?

**Reply by shishkin17** [reply]
Yes it does. 
I would leave this as it can be useful if somebody has the same problem.

**Reply by shishkin17** [reply]
> I'm not sure I understand what the problem was and this solves it?

I just came across a similar issue (so that you could understand better): https://github.com/vikejs/vike/issues/19.

I am talking about the same, but in terms of locales.

**Reply by brillout** [reply]
I don't understand why setting `noExtraDir` to the value you need isn't enough for you.

**Reply by shishkin17** [reply]
I don't know how to explain better than in the first message.

But I don't have a problem right now. I managed to get what I wanted with other manipulations, so no action is needed.
Maybe it is working as it is intended to work.
